### DISCUSSION TITLE
I have a problem with my Github account

### DISCUSSION THREAD

**Discussion by husanIbragimov** [community/community#121645] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Several month ago, I unknowingly turned on 2fa and did not scan the TOTP Qr code but everything was okey. Then I musted to install Ubuntu another version and I remove every my account from my computer browser because I know every password. So I done it. I installed new OS but I didn't access  to Github. Gihub send me recovery code from email before Github asked "Open your two-factor authenticator (TOTP) app or browser extension to view your authentication code."

I unknowingly sent "unlink email" message service on my email. Unlink came and I clicked on that. I tried to get help Virtual Assistant and GitHub support.

I tried to reset password but didn't accept that

![image](https://github.com/community/community/assets/134776990/67b59102-4d89-48e1-91de-9ed10f9c063f)

**Comment by ebndev** [comment]
Hi @ibragimovxusan,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
